From a423ddf98d423afeb14feb4136672c81a66d9240 Mon Sep 17 00:00:00 2001
From: "lowren.h.lawson@intel.com" <lowren.h.lawson@intel.com>
Date: Tue, 5 Oct 2021 13:08:17 -0700
Subject: [PATCH] Added nstream-cublas mpi init & finalize for GEOPM

Signed-off-by: lowren.h.lawson@intel.com <lowren.h.lawson@intel.com>
---
 Cxx11/nstream-cublas.cu | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Cxx11/nstream-cublas.cu b/Cxx11/nstream-cublas.cu
index b858b17c..bc9a56e1 100644
--- a/Cxx11/nstream-cublas.cu
+++ b/Cxx11/nstream-cublas.cu
@@ -64,9 +64,11 @@
 
 #include "prk_util.h"
 #include "prk_cuda.h"
+#include "prk_mpi.h"
 
 int main(int argc, char * argv[])
 {
+  prk::MPI::state mpi(&argc,&argv);
   std::cout << "Parallel Research Kernels version " << PRKVERSION << std::endl;
   std::cout << "C++11/CUBLAS STREAM triad: A = B + scalar * C" << std::endl;
 
-- 
2.18.1

