From 8130312a0fc15ded95c321ed900f268b0e06e94a Mon Sep 17 00:00:00 2001
From: "lowren.h.lawson@intel.com" <lowren.h.lawson@intel.com>
Date: Tue, 5 Oct 2021 14:21:54 -0700
Subject: [PATCH 2/3] Added sgemm-cublas mpi init & finalize for GEOPM

Signed-off-by: lowren.h.lawson@intel.com <lowren.h.lawson@intel.com>
---
 Cxx11/sgemm-cublas.cu | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Cxx11/sgemm-cublas.cu b/Cxx11/sgemm-cublas.cu
index 8263b44f..27a2f6f2 100644
--- a/Cxx11/sgemm-cublas.cu
+++ b/Cxx11/sgemm-cublas.cu
@@ -62,6 +62,7 @@
 
 #include "prk_util.h"
 #include "prk_cuda.h"
+#include "prk_mpi.h"
 
 #if 0
 __global__ void init(unsigned order, float * A, float * B, float * C)
@@ -172,6 +173,7 @@ void prk_bgemm(const cublasHandle_t & h,
 
 int main(int argc, char * argv[])
 {
+  prk::MPI::state mpi(&argc,&argv);
   std::cout << "Parallel Research Kernels version " << PRKVERSION << std::endl;
   std::cout << "C++11/CUBLAS Dense matrix-matrix multiplication: C += A x B" << std::endl;
 
-- 
2.18.1

